normandy/control/static/control/js/actions/ControlActions.js

```diff
@@ -98,11 +105,14 @@ function requestInProgress() {
   };
 }
 
-function requestComplete(status) {
-  return {
-    type: REQUEST_COMPLETE,
-    status: status
-  };
+function requestComplete(result) {
+  return (dispatch) => {
```

Comment:
I don't have a preference, as long as we're consistent. I feel like if we have to use parens when there are two or more params, we might as well use parens for all of them.